starport/templates/app/stargate/x/{{appName}}/client/rest/rest.go.plush

```diff
@@ -12,6 +13,14 @@ const (
 
 // RegisterRoutes registers <%= AppName %>-related REST handlers to a router
 func RegisterRoutes(clientCtx client.Context, r *mux.Router) {
-    // this line is used by starport scaffolding
+    // this line is used by starport scaffolding # 2
+}
+
+func registerQueryRoutes(clientCtx client.Context, r *mux.Router) {
```

Comment:
this was being created for each `type` which is error prone. instead, now we add handlers inside